js/utils.js

```diff
@@ -339,7 +525,8 @@ Fliplet.Registry.set('dynamicListUtils', function() {
     },
     Records: {
       runFilters: runRecordFilters,
-      getFields: getRecordFields
+      getFields: getRecordFields,
```

Comment:
@tonytlwu `getRecordFields` does not exist anywhere in these changes.

Reply:
@hcarneiro It's [here](https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/f34ea68aa7d09fe9a98496e4378667e4fb2cd439/js/utils.js#L279).

It's not in the changes because no changes were made to the function. I only added a `,` at the end.